regulations/views/comment.py

```diff
@@ -70,3 +71,29 @@ def submit_comment(request):
         'status': 'submitted',
         'metadata_url': metadata_url,
     })
+
+
+@csrf_exempt
+@require_http_methods(['GET', 'HEAD'])
+def get_federal_agencies(request):
+    url = "{}?{}".format(settings.REGS_GOV_API_LOOKUP_URL,
+                         "field=gov_agency&dependentOnValue=Federal")
+    response = requests.get(
+        url,
+        headers={'X-Api-Key': settings.REGS_GOV_API_KEY}
+    )
+    response.raise_for_status()
```

Comment:
No, I think it's a 500--django doesn't know how to inspect the `HTTPError` from requests:

<img width="899" alt="screen shot 2016-03-29 at 7 03 52 pm" src="https://cloud.githubusercontent.com/assets/1633460/14126727/19783bd8-f5e1-11e5-986a-7764520f554b.png">